tests/tests/market_tests.cpp

```diff
@@ -1195,9 +1202,11 @@ BOOST_AUTO_TEST_CASE(hard_fork_343_cross_test)
  */
 BOOST_AUTO_TEST_CASE(target_cr_test_limit_call)
 { try {
-   auto mi = db.get_global_properties().parameters.maintenance_interval;
-   generate_blocks(HARDFORK_CORE_834_TIME - mi);
-   generate_blocks(db.get_dynamic_global_properties().next_maintenance_time);
+
+   if(hf1270)
+      generate_blocks(HARDFORK_CORE_1270_TIME);
+   else
+      generate_blocks(HARDFORK_CORE_834_TIME);
```

Comment:
To be more accurate, here we should generate blocks until the maintenance time but not the hf time.